params/local.yml

```diff
@@ -2,6 +2,7 @@ reads: "${baseDir}/data/full_data/Myc_Raw_Data/*_R{1,2}_001.fastq.gz"
 outdir: "${baseDir}/results"
 gbk_file: "${baseDir}/data/full_data/NZ_LT906469.1.fasta"
 orthoani_jar: "${baseDir}/resources/OAT_cmd.jar"
+orthoani_fastas = "${baseDir}/data/final_fasta/*fasta"
```

Comment:
@abhi18av how do i proceed with the fix for this small typo?

Reply:
Open a PR 😉